bucoffea/monojet/definitions.py

```diff
@@ -438,16 +438,14 @@ def setup_candidates(df, cfg):
         hadflav= 0*df['Jet_pt'] if df['is_data'] else df['Jet_hadronFlavour'],
     )
 
-    # Only fur UL v8 samples, the new HF shape variables
-    if cfg.RUN.ULEGACYV8:
-        kwargs = {
-            'setaeta': df['Jet_hfsigmaEtaEta'],
-            'sphiphi': df['Jet_hfsigmaPhiPhi'],
-            'hfcentralstripsize': df['Jet_hfcentralEtaStripSize'],
-            'hfadjacentstripsize': df['Jet_hfadjacentEtaStripsSize'],
-            'btagdf': df['Jet_btagDeepFlavQG'],
-        }
-        ak4.add_attributes(**kwargs)
+    kwargs = {
+        'setaeta': df['Jet_hfsigmaEtaEta'],
+        'sphiphi': df['Jet_hfsigmaPhiPhi'],
+        'hfcentralstripsize': df['Jet_hfcentralEtaStripSize'],
+        'hfadjacentstripsize': df['Jet_hfadjacentEtaStripsSize'],
+        'btagdf': df['Jet_btagDeepFlavQG'],
+    }
+    ak4.add_attributes(**kwargs)
```

Comment:
Since there is no `if` statement anymore, should we just add these attributes to the `candidatesfromcounts` call just above?

Reply:
Make sense. I'll add it in the next commit.

Reply:
done